src/scripts/towns/TownList.ts

```diff
@@ -1934,7 +1934,7 @@ TownList['National Park'] = new Town(
     'National Park',
     GameConstants.Region.johto,
     GameConstants.JohtoSubRegions.Johto,
-    [new SafariTownContent(), JohtoContestShop],
+    [new SafariTownContent('Bug Catching Contest'), JohtoContestShop],
```

Comment:
It may be worth doing the same for the Friend Safari, as well?

Reply:
Sure. Want it to say just 'Friend Safari' or 'Enter Friend Safari'?

Reply:
I guess "Enter Friend Safari" to match the Kanto Safari?